cpp/CMakeLists.txt

```diff
@@ -376,6 +376,7 @@ add_library(
   src/io/json/legacy/json_gpu.cu
   src/io/json/legacy/reader_impl.cu
   src/io/json/write_json.cu
+  src/io/json/json_quote_normalization.cu
```

Comment:
Can we sort the filenames under src/io/json?

Reply:
Sorted!